cmake_modules/commonSetup.cmake

```diff
@@ -129,6 +129,8 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
   option(WSSQL_SERVICE "Configure use of ws_sql service" ON)
   option(USE_DIGISIGN "Use digisign" ON)
   option(INCLUDE_EE_PLUGINS "Install EE Plugins in Clienttool" OFF)
+  option(INCLUDE_TREEVIEW "Build legacy treeview" OFF)
```

Comment:
Shouldn't you be setting these in the action's yml file (to exclude from the win32 build test) rather than defaulting them off for all builds?

Reply:
Or maybe you are...

Reply:
Yes. I have for the others.  In this case I didn't think anyone would be likely to be using it/even know it existed.  @jakesmith should we delete the code?

Reply:
I don't think anyone is using it.
Opened JIRA to remove it: https://track.hpccsystems.com/browse/HPCC-24919